src/mngt/ocf_mngt_io_class.c

```diff
@@ -256,6 +256,26 @@ static int _ocf_mngt_io_class_validate_cfg(ocf_cache_t cache,
 		return -OCF_ERR_INVAL;
 	}
 
+	if (cfg->max_size < PARTITION_SIZE_MIN ||
+			cfg->max_size > PARTITION_SIZE_MAX) {
+		ocf_cache_log(cache, log_info,
+				"Invalid value of the partition max size\n");
+		return -OCF_ERR_INVAL;
+	}
+
+	if (cfg->min_size < PARTITION_SIZE_MIN ||
+			cfg->min_size > PARTITION_SIZE_MAX) {
+		ocf_cache_log(cache, log_info,
+				"Invalid value of the partition min size\n");
+		return -OCF_ERR_INVAL;
+	}
```

Comment:
Is the "min size" being validated? I think no, I would just remove this from IO class config.

Reply:
Ok, I removed it from the public API